discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
+            "userLoginWithToken": "yes",
+            "botLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "automatic",
+            "apiDocsUrl": "http://www.rubydoc.info/gems/discordrb",
+            "tutorials": "yes",
+            "tutorialsUrl": "https://github.com/meew0/discordrb/wiki",
+            "examples": "yes",
+            "examplesUrl": "https://github.com/meew0/discordrb/tree/master/examples",
+            "eventsAsync": "threaded",
```

Comment:
Here I went with @Rapptz's suggestion of four values:
- `async` for a lib that uses coroutine-based events
- `threaded` for a lib that uses threaded events, for example called using a thread pool or with a new thread each
- `thread-safe` for a lib that doesn't use threaded events by default but where it is possible for the user to do this
- `none` for a lib that doesn't allow asynchronous events in any way